### ISSUE TITLE
lib error in fiona on dev images

### ISSUE THREAD

**Issue by delgadom** [RhodiumGroup/docker_images#30]
@bolliger32 reports an issue on [worker:dev](https://hub.docker.com/r/rhodium/worker/) and [bolliger32-clawpack-image](https://hub.docker.com/r/climateimpactlab/bolliger32-clawpack-image/) calling a function from geopandas. error occurs inside fiona:

`ImportError: libkea.so.1.4.7: cannot open shared object file: No such file or directory`

@bolliger32 can you post a full stack trace here?

**Comment by bolliger32** [comment]
Here's what I get when submitting a function that just imports geopandas (or fiona):

```distributed.scheduler - ERROR - error from worker tcp://10.28.11.48:36153: libkea.so.1.4.7: cannot open shared object file: No such file or directory
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-3-0c382763d24b> in <module>()
      2     import geopandas
      3     return None
----> 4 client.submit(import_gpd).result()

/opt/conda/lib/python3.6/site-packages/distributed/client.py in result(self, timeout)
    169                                   raiseit=False)
    170         if self.status == 'error':
--> 171             six.reraise(*result)
    172         elif self.status == 'cancelled':
    173             raise result

/opt/conda/lib/python3.6/site-packages/six.py in reraise(tp, value, tb)
    690                 value = tp()
    691             if value.__traceback__ is not tb:
--> 692                 raise value.with_traceback(tb)
    693             raise value
    694         finally:

<ipython-input-3-0c382763d24b> in import_gpd()
      1 def import_gpd():
----> 2     import geopandas
      3     return None
      4 client.submit(import_gpd).result()

/opt/conda/lib/python3.6/site-packages/geopandas/__init__.py in <module>()
      4 from geopandas.geodataframe import GeoDataFrame, concat
      5 
----> 6 from geopandas.io.file import read_file
      7 from geopandas.io.sql import read_postgis
      8 from geopandas.tools import sjoin

/opt/conda/lib/python3.6/site-packages/geopandas/io/file.py in <module>()
      3 import os
      4 
----> 5 import fiona
      6 import numpy as np
      7 import six

/opt/conda/lib/python3.6/site-packages/fiona/__init__.py in <module>()
     67 from six import string_types
     68 
---> 69 from fiona.collection import Collection, BytesCollection, vsi_path
     70 from fiona._drivers import driver_count, GDALEnv
     71 from fiona.drvsupport import supported_drivers

/opt/conda/lib/python3.6/site-packages/fiona/collection.py in <module>()
      7 
      8 from fiona import compat
----> 9 from fiona.ogrext import Iterator, ItemsIterator, KeysIterator
     10 from fiona.ogrext import Session, WritingSession
     11 from fiona.ogrext import (

ImportError: libkea.so.1.4.7: cannot open shared object file: No such file or directory```

**Comment by delgadom** [comment]
fixed by #29
